libexec/scoop-download.ps1

```diff
@@ -57,7 +57,7 @@ foreach ($curr_app in $apps) {
     $app, $bucket, $version = parse_app $curr_app
     $app, $manifest, $bucket, $url = Get-Manifest "$bucket/$app"
 
-    info "Starting download for $app..."
+    info "Downloading '$app'$(if ($version) { " ($version)" }) [$architecture]$(if ($bucket) { " from $bucket bucket" })"
```

Comment:
`$version` will always exist, no?

Reply:
will only exist when a version is specified now. `scoop install <app>@version`

Reply:
I see